src/main/java/com/metamx/collections/bitmap/WrappedImmutableConciseBitmap.java

```diff
@@ -52,7 +53,10 @@ public ImmutableConciseSet getBitmap()
   @Override
   public boolean get(int value)
   {
-    return bitmap.get(value) > 0;
+    final ConciseSet conciseSet = new ConciseSet();
+    conciseSet.add(value);
+    final ImmutableConciseSet immutableConciseSet = ImmutableConciseSet.newImmutableFromMutable(conciseSet);
+    return ImmutableConciseSet.intersection(this.bitmap, immutableConciseSet).size() != 0;
```

Comment:
My benchmarks say that this iterator remains near similar to the previous one as it still extracts all ints from a CPU word. Results show that performing an intersection remains about 3 times faster than itertaing over bitmap's set bits. Link to benchmarks: https://bitbucket.org/samytto/bytebuffercollections_benchgetmethods/src/8b9aa5001bd992a0c019a0929c31de3e5749e303/src/test/java/com/metamx/collections/bitmap/BitmapBenchmark.java?at=master&fileviewer=file-view-default#BitmapBenchmark.java-142

My results:
data generated. Nb data = 100000
Running com.metamx.collections.bitmap.RangeBitmapBenchmarkTest
RangeBitmapBenchmarkTest.timeGetIterator: [measured 2 out of 3 rounds, threads: 1 (sequential)]
 round: 2.39 [+- 0.01], round.block: 0.00 [+- 0.00], round.gc: 0.00 [+- 0.00], GC.calls: 0, GC.time: 0.00, time.total: 7.23, time.warmup: 2.46, time.bench: 4.77
RangeBitmapBenchmarkTest.timeBufferGet: [measured 2 out of 3 rounds, threads: 1 (sequential)]
 round: 1.97 [+- 0.00], round.block: 0.00 [+- 0.00], round.gc: 0.00 [+- 0.00], GC.calls: 0, GC.time: 0.00, time.total: 5.92, time.warmup: 1.97, time.bench: 3.95
RangeBitmapBenchmarkTest.timeGetIntersect: [measured 2 out of 3 rounds, threads: 1 (sequential)]
 round: 0.54 [+- 0.00], round.block: 0.00 [+- 0.00], round.gc: 0.00 [+- 0.00], GC.calls: 0, GC.time: 0.00, time.total: 1.74, time.warmup: 0.66, time.bench: 1.09
data generated. Nb data = 100000
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 15.064 sec - in com.metamx.collections.bitmap.RangeBitmapBenchmarkTest
Running com.metamx.collections.bitmap.UniformBitmapBenchmarkTest
UniformBitmapBenchmarkTest.timeGetIterator: [measured 2 out of 3 rounds, threads: 1 (sequential)]
 round: 2.31 [+- 0.00], round.block: 0.00 [+- 0.00], round.gc: 0.00 [+- 0.00], GC.calls: 0, GC.time: 0.00, time.total: 6.96, time.warmup: 2.34, time.bench: 4.62
UniformBitmapBenchmarkTest.timeBufferGet: [measured 2 out of 3 rounds, threads: 1 (sequential)]
 round: 1.95 [+- 0.00], round.block: 0.00 [+- 0.00], round.gc: 0.00 [+- 0.00], GC.calls: 0, GC.time: 0.00, time.total: 5.85, time.warmup: 1.96, time.bench: 3.90
UniformBitmapBenchmarkTest.timeGetIntersect: [measured 2 out of 3 rounds, threads: 1 (sequential)]
 round: 0.54 [+- 0.00], round.block: 0.00 [+- 0.00], round.gc: 0.00 [+- 0.00], GC.calls: 0, GC.time: 0.00, time.total: 1.63, time.warmup: 0.55, time.bench: 1.08
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 14.418 sec - in com.metamx.collections.bitmap.UniformBitmapBenchmarkTest